crates/wasmtime/src/runtime/vm/vmcontext.rs

```diff
@@ -266,16 +266,20 @@ pub struct VMMemoryDefinition {
     /// atomically. For relaxed access, see
     /// [`VMMemoryDefinition::current_length()`].
     pub current_length: AtomicUsize,
+
+    /// The log2 of this memory's page size, in bytes.
+    pub page_size_log2: u8,
```

Comment:
How come this is stored here? This should be statically known from ambient context elsewhere I think? (e.g. this just stores the runtime state of the base/current byte length)

Reply:
I think I was using it for an approach that I ultimately abandoned.